src/main/drivers/lights_hal.c

```diff
@@ -0,0 +1,33 @@
+#include "drivers/lights_hal.h"
+#include "drivers/io.h"
+
+#ifdef USE_LIGHTS
+
+#if (!defined(LIGHTS_USE_PCA9685_OUTPUT)) && (!defined(LIGHTS_OUTPUT_MODE))
+    #define LIGHTS_OUTPUT_MODE IOCFG_OUT_PP
+#endif
+
+
+static IO_t lightsIO = DEFIO_IO(NONE);
+
+bool lightsHardwareInit()
+{
+#ifdef LIGHTS_USE_PCA9685_OUTPUT
```

Comment:
I think we can take PCA9685 out of the equation. As long as we have a place that handles hardware, it can just always write to pwm ports